frame/system/src/lib.rs

```diff
@@ -500,6 +501,11 @@ decl_error! {
 	}
 }
 
+/// Pallet struct placeholder on which is implemented the pallet logic.
+///
+/// It is currently an alias for `Module` as old macros still generate/use old name.
+pub type Pallet<T> = Module<T>;
```

Comment:
I presume we'll want to mark `Module<T>` deprecated, no?

Reply:
Yes I think we can mark deprecate in a PR which moves node runtimes usage of system Module and other usage to avoid to much warning on compilation